client/src/css/structure.css

```diff
@@ -888,19 +888,35 @@ a[disabled] {
   background-color: #ff000083 !important;
 }
 
+.icon-expired {
+  color: #ff000083 !important;
```

Comment:
I discovered it for the first time in this file, it is in several classes, this will force that the defined values have priority over any other value that will be defined

Reply:
The last two hex digits in the long format control the transparency of the item.  So '83' means Transparency should be 83/256 (or 32% transparent).

Reply:
See https://css-tricks.com/8-digit-hex-codes/